Header header
uint64 time_usec
uint8 system_id
float64 lat
float64 lon
float64 alt
float64 relative_alt
float64 heading